modules/seqexec/server/src/main/scala/seqexec/server/gmos/Gmos.scala

```diff
@@ -164,23 +166,23 @@ object Gmos {
   // In principle we'd expect the OT to send the sequence but instead the
   // sequence is hardcoded in the seqexec and we only read the positions from
   // the OT
-  val NsSequence = NonEmptyList.of(StageB, StageA, StageA, StageB)
+  val NsSequence: NonEmptyList[NodAndShuffleStage] = NonEmptyList.of(StageB, StageA, StageA, StageB)
```

Comment:
👍

Reply:
I cannot take the credit, Idea told me to do it :)